test/GateKeeperTest.ts

```diff
@@ -0,0 +1,93 @@
+import { Alchemy, Network } from "alchemy-sdk"
+import { expect } from "chai"
+import { ethers } from "hardhat"
+import { TicketAddress } from "../constants/Ticket"
+import { Keypair } from "qaci-domainobjs"
+import { TestMACI, TestMACI__factory, SignUpNFTGatekeeper, SignUpNFTGatekeeper__factory } from "../typechain-types"
+
+const provider = ethers.getDefaultProvider()
+const settings = {
+  apiKey: `${process.env.ALCHEMY_API_KEY}`,
+  network: Network.OPT_MAINNET,
+}
+
+const holders: string[] = [
+  "0xe9a8A1F356bD457d33C31730F1Dbea8038835bED",
+  "0x7Bb3Db8429059684Bf2cEb01861E6438B4b99FC0",
+  "0x322Ebd2Aa9Df8f33cc2010AF5375aFCdDe54397e",
+  "0x2C8d1FD63aa75F1B8b5b2380dA75D7Ee333C6db0",
+  "0xd1fFdA9C225DDEE34f0837BF4D4a441bDd54C473",
+  "0x3779cD041672a4bf2AeDc5bEf1320940a1baC89A",
+  "0x22a274A520587d979436Bde4D7495137a25f0044",
+  "0xD63fA8eaF82078Eb2E1515a5d892AD3B3bEa32FA",
+]
+
+const notHolder: string = "0x16c4AAc26A4171a9873297cd0c649808f8E36f30"
```

Comment:
```suggestion
const notHolder: string = "0x4200000000000000000000000000000000000010"
```

`0x16c4AAc26A4171a9873297cd0c649808f8E36f30` 는 nft 보유주소라서 테스트가 아래처럼 깨지길래 옵티미즘 브리지 주소로 바꿨어요!:
```
 4 passing (24s)
  1 failing

  1) SignUp
       Fail if don't have NFT ticket:

      AssertionError: expected 1 to deeply equal +0
      + expected - actual

      -1
      +0
      
      at Context.<anonymous> (test/GateKeeperTest.ts:66:43)
      at processTicksAndRejections (node:internal/process/task_queues:95:5)
```